.circleci/unittest/linux/scripts/install.sh

```diff
@@ -37,7 +33,8 @@ printf "Installing PyTorch with %s\n" "${cudatoolkit}"
 if [ "${os}" == "MacOSX" ]; then
     conda install -y -c "pytorch-${UPLOAD_CHANNEL}" "pytorch-${UPLOAD_CHANNEL}"::pytorch "${cudatoolkit}"
 else
-    conda install -y -c "pytorch-${UPLOAD_CHANNEL}" -c nvidia "pytorch-${UPLOAD_CHANNEL}"::pytorch[build="*${version}*"] "${cudatoolkit}"
+    printf "conda install -y pytorch ${cudatoolkit} -c pytorch-${UPLOAD_CHANNEL} -c nvidia"
+    conda install -y  pytorch "${cudatoolkit}" -c "pytorch-${UPLOAD_CHANNEL}" -c nvidia
```

Comment:
This does not seem to be equivalent to the previous statement (i.e. it remove exact pytorch version contraint)

Reply:
I don't think we need it here honestly. I think we better stick to simpler solution same as get started page. Would be easier to maintain

Reply:
yes totally old install command would yield:
```
conda install -y -c pytorch-nightly -c nvidia -c pytorch-nightly::pytorch[build="*cu117*"] pytorch-cuda=11.7
```
new one is the same as in our get started page:
```
conda install pytorch pytorch-cuda=11.7 -c pytorch-nightly -c nvidia
```

Reply:
This is unit test if it fails means we have real problem with pytorch binary